src/commands/admin/refresh.ts

```diff
@@ -42,14 +42,14 @@ export default class extends Command {
 				description: command.description,
 				options: command?.options || [],
 				defaultPermission: false
-			});
+			} as ApplicationCommandData);
```

Comment:
I've removed the typecast as ApplicationCommandData, would I still need the ``defaultPermission: false`` to be there or can I remove it as without as ApplicationCommandData it will error suggesting: **"Argument of type '{ name: string; description: string; options: ApplicationCommandOptionData[]; defaultPermission: boolean; }' is not assignable to parameter of type 'ApplicationCommandData'.
  Object literal may only specify known properties, and 'defaultPermission' does not exist in type 'ApplicationCommandData'."**?